### PR TITLE
[Docs] Translate installation and 15_min

### PR DESCRIPTION
**Pull request by xin-li-67**
## Motivation

Translate documents

## Modification

docs/en/get_started/15_minutes.md
docs/en/get_started/installation.md

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/en/get_started/installation.md [hunk 1]
```diff
@@ -1,3 +1,75 @@
 ## Installation
```
Comment (zhouzaida) [inline docs/en/get_started/installation.md:1]:
```suggestion
# Installation
```

File: docs/en/get_started/installation.md [hunk 2]
```diff
@@ -1,3 +1,75 @@
...
 
-Coming soon. Please refer to [chinese documentation](https://mmengine.readthedocs.io/zh_CN/latest/get_started/installation.html).
+### Dependencies
```
Comment (zhouzaida) [inline docs/en/get_started/installation.md:3]:
```suggestion
## Prerequisites
```

File: docs/en/get_started/installation.md [hunk 3]
```diff
@@ -1,3 +1,75 @@
...
+
+2. Install PyTorch
+
+   Before installing `MMEngine`, please make sure that PyTorch has been successfully installed in the environment, you can refer to [PyTorch official installation documentation](https://pytorch.org/get-started/locally/#start-locally). Verify if PyTorch is installed with the following command:
+
+   ```bash
+   python -c 'import torch;print(torch.__version__)'
+   ```
+
+### Install MMEngine
```
Comment (zhouzaida) [inline docs/en/get_started/installation.md:27]:
```suggestion
## Install MMEngine
```

File: docs/en/get_started/installation.md [hunk 4]
```diff
@@ -1,3 +1,75 @@
...
+
+#### Install with mim
```
Comment (zhouzaida) [inline docs/en/get_started/installation.md:29]:
```suggestion
### Install with mim
```

File: docs/en/get_started/installation.md [hunk 5]
```diff
@@ -1,3 +1,75 @@
...
+
+[mim](https://github.com/open-mmlab/mim) is a package management tool for OpenMMLab projects, which can be used to install the OpenMMLab project easily.
+
+```bash
+pip install -U openmim
+mim install mmengine
+```
+
+#### Install with pip
```
Comment (zhouzaida) [inline docs/en/get_started/installation.md:38]:
```suggestion
### Install with pip
```

File: docs/en/get_started/installation.md [hunk 6]
```diff
@@ -1,3 +1,75 @@
...
+
+```bash
+pip install mmengine
+```
+
+#### Use docker images
+
+1. build the image
```
Comment (zhouzaida) [inline docs/en/get_started/installation.md:46]:
```suggestion
1. Build the image
```

File: docs/en/get_started/installation.md [hunk 7]
```diff
@@ -1,3 +1,75 @@
...
+
+   ```bash
+   docker build -t mmengine https://github.com/open-mmlab/mmengine.git#main:docker/release
+   ```
+
+   More information can be referred from [mmengine/docker](https://github.com/open-mmlab/mmengine/tree/main/docker).
+
+2. run the image
```
Comment (zhouzaida) [inline docs/en/get_started/installation.md:54]:
```suggestion
2. Run the image
```

File: docs/en/get_started/15_minutes.md [hunk 8]
```diff
@@ -1,3 +1,241 @@
...
+1. [Build a Model](#build-a-model)
+2. [Build a Dataset and DataLoader](#build-a-dataset-and-dataloader)
+3. [Build a Evaluation Metrics](#build-a-evaluation-metrics)
+4. [Build a Runner and Run the Task](#build-a-runner-and-run-the-task)
+
+## Build a Model
+
+First, we need to build a **model**. In MMEngine, the model should inherit `BaseModel`. Besides accepting several parameters from the dataset, its `forward` method needs to accept an extra argument called `mode`:
+
+- for training, the value of `mode` is "loss," and the `forward` method should return a   `dict` containing the key "loss".
```
Comment (zhouzaida) [inline docs/en/get_started/15_minutes.md:15]:
```suggestion
- for training, the value of `mode` is "loss," and the `forward` method should return a `dict` containing the key "loss".
```

File: docs/en/get_started/15_minutes.md [hunk 9]
```diff
@@ -1,3 +1,241 @@
...
+The whole process includes the following steps:
+
+1. [Build a Model](#build-a-model)
+2. [Build a Dataset and DataLoader](#build-a-dataset-and-dataloader)
+3. [Build a Evaluation Metrics](#build-a-evaluation-metrics)
+4. [Build a Runner and Run the Task](#build-a-runner-and-run-the-task)
+
+## Build a Model
+
+First, we need to build a **model**. In MMEngine, the model should inherit `BaseModel`. Besides accepting several parameters from the dataset, its `forward` method needs to accept an extra argument called `mode`:
```
Comment (zhouzaida) [inline docs/en/get_started/15_minutes.md:13]:
```suggestion
First, we need to build a **model**. In MMEngine, the model should inherit from `BaseModel`. Besides accepting several parameters from the dataset, its `forward` method needs to accept an extra argument called `mode`:
```

File: docs/en/get_started/15_minutes.md [hunk 10]
```diff
@@ -1,3 +1,241 @@
...
+                                transform=transforms.Compose([
+                                    transforms.ToTensor(),
+                                    transforms.Normalize(**norm_cfg)
+                                ])))
+```
+
+## Build a Evaluation Metrics
+
+To validate and test the model, we need to define a **Metric** called accuracy to evaluate the model. This metric needs inherit from `BaseMetric` and implements the `process` and `compute_metrics` methods where the `process` method accepts the output of the dataset and other outputs when `mode="predict"`. The output data at this scenario is a batch of data. After processing this batch of data, we save the information to `self.results` property.
+`compute_metrics` accepts a `results` parameter. The input of this parameter is all the information saved in `process` (In the case of a distributed environment, `results` are the information collected from all `process` in all the threads). Using these information to calculate and return a `dict` that holds the results of the evaluation metrics
```
Comment (zhouzaida) [inline docs/en/get_started/15_minutes.md:75]:
```suggestion
`compute_metrics` accepts a `results` parameter. The input `results` of `compute_metrics` is all the information saved in `process` (In the case of a distributed environment, `results` are the information collected from all `process` in all the processes). Using these information to calculate and return a `dict` that holds the results of the evaluation metrics
```

**Comment by zhouzaida** [comment]
Hi @xin-li-67 , thanks for your contribution.

File: docs/en/get_started/installation.md [hunk 11]
```diff
@@ -1,3 +1,75 @@
...
+mim install mmengine
+```
+
+### Install with pip
+
+```bash
+pip install mmengine
+```
+
+#### Use docker images
```
Comment (zhouzaida) [inline docs/en/get_started/installation.md:44]:
```suggestion
### Use docker images
```

**Review by zhouzaida — APPROVED** [review]

File: docs/en/get_started/15_minutes.md [hunk 12]
```diff
@@ -1,3 +1,241 @@
 # 15 minutes to get started with MMEngine
 
-Coming soon. Please refer to [chinese documentation](https://mmengine.readthedocs.io/zh_CN/latest/get_started/15_minutes.html).
+Take training a ResNet-50 model on CIFAR-10 dataset as an example, we could build a complete, and configurable process for both training and validation in only 80 lines of code with `MMEgnine`.
```
Comment (C1rN09) [inline docs/en/get_started/15_minutes.md:3]:
```suggestion
In this tutorial, we'll take training a ResNet-50 model on CIFAR-10 dataset as an example. We will build a complete and configurable pipeline for both training and validation in only 80 lines of code with `MMEgnine`.
```

File: docs/en/get_started/15_minutes.md [hunk 13]
```diff
@@ -1,3 +1,241 @@
...
+The whole process includes the following steps:
+
+1. [Build a Model](#build-a-model)
+2. [Build a Dataset and DataLoader](#build-a-dataset-and-dataloader)
+3. [Build a Evaluation Metrics](#build-a-evaluation-metrics)
+4. [Build a Runner and Run the Task](#build-a-runner-and-run-the-task)
+
+## Build a Model
+
+First, we need to build a **model**. In MMEngine, the model should inherit from `BaseModel`. Besides accepting several parameters from the dataset, its `forward` method needs to accept an extra argument called `mode`:
```
Comment (C1rN09) [inline docs/en/get_started/15_minutes.md:13]:
```suggestion
First, we need to build a **model**. In MMEngine, the model should inherit from `BaseModel`. Aside from parameters representing inputs from the dataset, its `forward` method needs to accept an extra argument called `mode`:
```

File: docs/en/get_started/15_minutes.md [hunk 14]
```diff
@@ -1,3 +1,241 @@
...
+                                transform=transforms.Compose([
+                                    transforms.ToTensor(),
+                                    transforms.Normalize(**norm_cfg)
+                                ])))
+```
+
+## Build a Evaluation Metrics
+
+To validate and test the model, we need to define a **Metric** called accuracy to evaluate the model. This metric needs inherit from `BaseMetric` and implements the `process` and `compute_metrics` methods where the `process` method accepts the output of the dataset and other outputs when `mode="predict"`. The output data at this scenario is a batch of data. After processing this batch of data, we save the information to `self.results` property.
+`compute_metrics` accepts a `results` parameter. The input `results` of `compute_metrics` is all the information saved in `process` (In the case of a distributed environment, `results` are the information collected from all `process` in all the processes). Using these information to calculate and return a `dict` that holds the results of the evaluation metrics
```
Comment (C1rN09) [inline docs/en/get_started/15_minutes.md:75]:
```suggestion
`compute_metrics` accepts a `results` parameter. The input `results` of `compute_metrics` is all the information saved in `process` (In the case of a distributed environment, `results` are the information collected from all `process` in all the processes). Use these information to calculate and return a `dict` that holds the results of the evaluation metrics
```

File: docs/en/get_started/installation.md [hunk 15]
```diff
@@ -1,3 +1,75 @@
...
+1. Use conda and activate the environment:
+
+   ```bash
+   conda create -n open-mmlab python=3.7 -y
+   conda activate open-mmlab
+   ```
+
+2. Install PyTorch
+
+   Before installing `MMEngine`, please make sure that PyTorch has been successfully installed in the environment, you can refer to [PyTorch official installation documentation](https://pytorch.org/get-started/locally/#start-locally). Verify if PyTorch is installed with the following command:
```
Comment (C1rN09) [inline docs/en/get_started/installation.md:21]:
```suggestion
   Before installing `MMEngine`, please make sure that PyTorch has been successfully installed in the environment. You can refer to [PyTorch official installation documentation](https://pytorch.org/get-started/locally/#start-locally). Verify the installation with the following command:
```

**Comment by C1rN09** [comment]
Hi, thanks for your contribution! As a reminder, we are currently working on a refine of some Chinese documentations. The range covers **tutorials**, **advance tutorials** and **architecture design**. Since the above mentioned docs are **due to change**, you might translate other docs if you are willing to continue your contribution ^_^

**Comment by C1rN09** [comment]
Another thing, we recommend merging into `main` branch directly in MMEngine. Could you change the destination?

**Review by C1rN09 — APPROVED** [review]
LGTM
